kickoff/views/submit.py

```diff
@@ -8,15 +8,38 @@
 from kickoff.model import getReleaseTable
 from kickoff.views.forms import FennecReleaseForm, FirefoxReleaseForm, \
     ThunderbirdReleaseForm, getReleaseForm
+import urllib2
+import json
 
 log = logging.getLogger(__name__)
 
 
+def retrieveADI():
+    defaultURL = "http://localhost/"
```

Comment:
We won't be hosting this file on ship-it.mozilla.org. The way it's deployed doesn't allow for it. This probably needs to be defined somewhere in the js.